### PR TITLE
CFIN-469 Add new collections to documentation

### PR DESCRIPTION
**Pull request by RichardAllum**
Some updates with the new changes from CFIN-237

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ The [latest version of the API specification](https://university-of-york.github.
 
 A Funnelback collection is a set of data that can be searched. It is possible for collections to contain other collections, as the `york-uni-campusmap` collection contains others.
 
+There are three collections for courses.
+- `york-uni-courses` is powered by a web crawler that indexes all York courses.
+- `external-courses` is a [custom collection](https://docs.squiz.net/funnelback/archive/collections/collection-types/custom.html) 
+  that contains
```
Comment (ncn1) [inline README.md:30]: Missing details of what it contains...

File: spec/openAPI.yml [hunk 2]
```diff
@@ -42,6 +42,7 @@ paths:
             example: york-uni-courses
             enum:
               - york-uni-courses
```
Comment (ce666-york) [inline spec/openAPI.yml:44]: I think we should remove this as an option - even though it's technically possible, we're not looking to use it in this way from now on

**Review by ce666-york — COMMENTED** [review]
I think it would be good to rephrase the testing/deployment sections that mention `york-uni-courses` explicitly to say something more generic like "the collection you're looking to configure"

File: README.md (hunk 1 above)
Comment (RichardAllum) [inline README.md:30]:
```suggestion
- `external-courses` is a [custom collection](https://docs.squiz.net/funnelback/archive/collections/collection-types/custom.html) 
  that contains details on HYMS courses published by the CMS.
```

File: spec/openAPI.yml (hunk 2 above)
Comment (RichardAllum) [inline spec/openAPI.yml:44]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -24,6 +24,12 @@ The [latest version of the API specification](https://university-of-york.github.
...
+- `courses` is a [meta collection](https://docs.squiz.net/funnelback/archive/collections/collection-types/meta/index.html) that combines both of the other collections.
+
 The course collection is called `york-uni-courses`.
```
Comment (RichardAllum) [inline README.md:33]:
```suggestion
The course collection we use for the courses API is `courses`.
```

File: spec/openAPI.yml [hunk 4]
```diff
@@ -42,6 +42,7 @@ paths:
             example: york-uni-courses
```
Comment (RichardAllum) [inline spec/openAPI.yml:42]:
```suggestion
            example: courses
```

**Pull request merged**